### ISSUE TITLE
Quotes in custom metric is breaking Save button in Question Builder

### ISSUE THREAD

**Issue by salsakran** [metabase/metabase#11054]
**Describe the bug**
From a user (metric name redacted).
```
We have a custom metric without a name: Sum("Blah Official  Min") / Sum("Blah Window Min")

When clicking on Save, it does nothing and the following exception is thrown in the browser console. It works fine if you give the metric a name, so I think it's due to the quotes. 
```

We are running the latest Enterprise version.
**Logs**

Uncaught Error: `query` is a required parameter to format expressions
    at app-main.bundle.js?0ee80afa6574533daf17:5
    at s (app-main.bundle.js?0ee80afa6574533daf17:5)
    at app-main.bundle.js?0ee80afa6574533daf17:5
    at Array.map (<anonymous>)
    at s (app-main.bundle.js?0ee80afa6574533daf17:5)
    at app-main.bundle.js?0ee80afa6574533daf17:5
    at Array.map (<anonymous>)
    at s (app-main.bundle.js?0ee80afa6574533daf17:5)
    at app-main.bundle.js?0ee80afa6574533daf17:5
    at Array.map (<anonymous>)

**Comment by flamber** [comment]
@salsakran Looks like a duplicate of #10985 - it's not the quotes causing the problem, but the math.

**Comment by paulrosenzweig** [comment]
Fixed in #11065
